src/main/java/io/confluent/kafkarest/DefaultKafkaRestContext.java

```diff
@@ -20,15 +20,17 @@
 
 /**
  * Shared, global state for the REST proxy server, including configuration and connection pools.
+ * The objects for producer, admin and new Consumer would be lazy initialized.
```

Comment:
maybe 'ProducerPool, AdminClientWrapper and KafkaConsumerManager instances are initialized lazily'.